helm/csi-isilon/values.yaml

```diff
@@ -332,3 +332,8 @@ podmon:
   #    - "--mode=node"
   #    - "--leaderelection=false"
   #    - "--driver-config-params=/csi-isilon-config-params/driver-config-params.yaml"
+
+# maxPathLen: this parameter is used for setting the maximum Path length for the given volume.
+# Deafult value: 128
```

Comment:
typo

Reply:
update the go.mod to pick the latest gocsi with this change

Reply:
Done